### PR TITLE
Submission for Assignment 6

### PR DESCRIPTION
**Pull request by Chasemoudry**

### CODE REVIEW

**Comment by alex-g-carpenter** [comment]
poggers brah

File: what_feels_right/A.6_calc.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Calculator:
+    def __init__(self):
+        pass
+
+    def summ(self, a, b):
+        return a + b
```
Comment (colinrlly) [inline what_feels_right/A.6_calc.py:6]: mm the one liner sauce

File: what_feels_right/A.6_calc.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    def diff(self, a, b):
+        return a - b
+
+    def prod(self, a, b):
+        return a * b
+
+    def quot(self, a, b):
```
Comment (colinrlly) [inline what_feels_right/A.6_calc.py:14]: This stands for quotient 👍

**Review by colinrlly — APPROVED** [review]
Yes this is a class 👍

**Pull request merged**